src/mixins/button/_emphasized.scss

```diff
@@ -23,11 +23,15 @@
 }
 
 @mixin emphasizedPressedFocus() {
-  outline-color: var(--sapContent_ContrastFocusColor);
+  &::after {
```

Comment:
Maybe it's worth introducing so we can more easily control element used as focus host?

```
@mixin buttonFocusLocation() {
  &::after {
    @content
  }
}
```

Reply:
Let's postpone it to buttons refactor

Reply:
Aren't we refactoring Buttons in this PR? I think we should try to finalize the Button refactoring and stop introducing Breaking Changes for this component.

Reply:
It has been addressed - I removed some redundant mixins, which didn't bring any value